tests/data/test_historical_option_data.py

```diff
@@ -0,0 +1,774 @@
+import urllib.parse
+from datetime import datetime, timezone
+from typing import Dict
+
+from alpaca.data import Quote, Snapshot, Trade
+from alpaca.data.enums import DataFeed, Exchange
+from alpaca.data.historical.option import OptionHistoricalDataClient
+from alpaca.data.models import QuoteSet, TradeSet
+from alpaca.data.requests import (
+    OptionLatestQuoteRequest,
+    OptionLatestTradeRequest,
+    OptionQuotesRequest,
+    OptionSnapshotRequest,
+    OptionTradesRequest,
+)
+
+
+def test_get_quotes(reqmock, option_client: OptionHistoricalDataClient):
+    # Test single symbol request
+
+    symbol = "AAPL240126P00050000"
+    start = datetime(2024, 1, 24)
+    limit = 2
+
+    _start_in_url = urllib.parse.quote_plus(
+        start.replace(tzinfo=timezone.utc).isoformat()
+    )
+
+    reqmock.get(
+        f"https://data.alpaca.markets/v1beta1/options/quotes?symbols={symbol}&start={_start_in_url}&limit={limit}",
+        text="""
+    {
+        "quotes": [
+            {
+                "t": "2024-01-24T09:00:00.000059Z",
+                "ax": "K",
+                "ap": 158.65,
+                "as": 1,
+                "bx": "Q",
+                "bp": 159.52,
+                "bs": 4,
+                "c": [
+                    "R"
+                ],
+                "z": "C"
```

Comment:
This is not an option response. Same is true many times in this file.

Reply:
Thank you! fixed responses for unit test cases.